src/stories/pages/Experiment/Experiment.js

```diff
@@ -82,6 +80,20 @@ export const Experiment = ({
     if (photoStepCompleted) {
       setShowNext(true);
       setCompletedSteps((prevState) => [...prevState, 3]);
+    } else if (completedSteps.includes(3)) {
+      // In the case that the participant's photo has passed face detection but
+      // later on uploads another photo that doesn't pass, we need to disable
+      // the 'Next' button again.
+      setShowNext(false);
+      setCompletedSteps((prevState) => {
+        // Remove `3` from the completed steps array
+        const prevCompletedSteps = [...prevState];
+        const index = prevCompletedSteps.indexOf(3);
+        if (index > -1) {
+          prevCompletedSteps.splice(index, 1);
+        }
+        return prevCompletedSteps;
+      });
```

Comment:
This should be fine as long as `photoStepCompleted` is set to `false` correctly whenever we need to get into the `else if`.